src/Summoner/Tui/Kit.hs

```diff
@@ -47,6 +57,17 @@ data Tool
     | Stack
     deriving (Show)
 
+data GitHub = GitHub
+    { gitHubAble     :: Able
+    , gitHubPrivate  :: Bool
+    , gitHubTravis   :: Bool
+    , gitHubAppVeyor :: Bool
+    } deriving (Show)
+
+data Able
+    = Enable
+    | Disable
```

Comment:
Is'nt this `Bool`? 🤔

Reply:
:thinking: Indeed. `radioField` API allows to specify `Bool` easily. Can-can.